ext/js/language/translator.js

```diff
@@ -2022,6 +2030,31 @@ export class Translator {
         databaseEntries.sort(compareFunction);
     }
 
+    /**
+     * @param {import('dictionary').KanjiDictionaryEntry[]} dictionaryEntries
+     */
+    _sortKanjiDictionaryEntries(dictionaryEntries) {
+        /**
+         * @param {import('dictionary').KanjiDictionaryEntry} v1
+         * @param {import('dictionary').KanjiDictionaryEntry} v2
+         * @returns {number}
+         */
+        const compareFunction = (v1, v2) => {
+            // Sort by dictionary priority
+            let i = v2.dictionaryPriority - v1.dictionaryPriority;
+            if (i !== 0) { return i; }
+
+            // Sort by dictionary order
+            i = v1.dictionaryIndex - v2.dictionaryIndex;
+            if (i !== 0) { return i; }
+
+            // Sort by definition count
+            i = v2.definitions.length - v1.definitions.length;
```

Comment:
IMO this would cause confusion if it led to different sorting behavior on different entries

Reply:
I'll remove this since this sorting condition couldn't be reached.